src/World.cpp

```diff
@@ -691,9 +696,61 @@ void cWorld::GenerateRandomSpawn(void)
 		}
 	}  // for i - 100*
 
-	m_SpawnY = (double)GetHeight((int)m_SpawnX, (int)m_SpawnZ) + 1.6f;  // 1.6f to accomodate player height
+	if (foundSpawnPoint)
+	{
+		m_SpawnY = (double)GetHeight((int)m_SpawnX, (int)m_SpawnZ) + 1.6f;  // 1.6f to accomodate player height
+
+		LOGINFO("Generated random spawnpoint position at {%i, %i, %i}", (int)m_SpawnX, (int)m_SpawnY, (int)m_SpawnZ);
+	}
+	else
+	{
+		LOGINFO("Did not find an acceptable spawnpoint. Generated a random spawnpoint position at {%i, %i, %i}", (int)m_SpawnX, (int)m_SpawnY, (int)m_SpawnZ);
+	} // Maybe widen the search instead?
```

Comment:
You need an extra space before starting the comment. That's why the build is failing. `}<space><space>// Mayb....`